Python/Setup/swix/Workloads/Microsoft.VisualStudio.Workload.Python.swixproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="SetupProjectBefore.settings" />
+
+  <ItemGroup>
+    <Package Include="$(MSBuildProjectName).res.swr" />
+
+    <Dependency Include="Microsoft.PythonTools" />
+    <Dependency Include="Microsoft.PythonTools.Web" />
+    <Dependency Include="Microsoft.PythonTools.Samples">
+      <Type>Optional</Type>
+    </Dependency>
+    <Dependency Include="Microsoft.PythonTools.Uwp">
```

Comment:
Doesn't ptvs also have dependencies on team explorer, web tools, and the like? is there a reason those aren't included in this workload file as hidden dependencies?